modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+Setting `inline_boundaries: false` turns this highlighting off.
+
+With this release, Real-time Collaboration now supports this setting.
+
+NOTE: the equivalent behavior could be achieved by setting `inline_boundaries_selector` to an empty string. And this setting was, previously supported by Real-time Collaboration (and still is). But setting `inline_boundaries` to `false` is the preferred method of turning inline element highlighting off.
+
+==== Lists selected with Select All (Ctrl+A or Cmd+A) did not provide a valid fragment for `tinymce.activeEditor.selection.getContent()`
+
+If a {productname} document contained only a list, and the entire document was selected using *Edit > Select All* (or by typing Ctrl+A or Cmd+A), the selection did not present as valid to `tinymce.activeEditor.selection.getContent()`.
+
+If `getContent` was run after such a document was selected using *Select All*, `getContent` returned an empty string.
+
+With this update, this behavior has been corrected. Selecting all of such a document and running `getContent` now returns an html list, as expected.
+
+==== The ‘reconnect’ message did not appear after connectivity was lost
+
+Previously, Real-time Collaboration only checked for jitter (the variations in the speed of signal transmission to signal reception over the network connection over time). Consequently, if a Network connection went down, and stayed down, the ‘connection lost’ alert did not appear.
+
+With this update, Real-time Collaboration now also checks for straight-up connectivity. And, if the connection goes down for 30 seconds, the ‘connection lost’ alert appears.
+
+For information on the **Real-Time Collaboration** plugin, see: xref:real-timecollaboration.adoc[Real-Time Collaboration plugin].
+
+
+=== Spell Checker Pro 3.0.3
+
+The {productname} vnumtxt release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 3.0.3 includes the following bug fix:
+
+==== An issue with the `DomDescent.toLeaf` API could result in Spell Checker Pro’s ‘preserve selection’ logic setting the selection when it did not need to
+
+The `DomDescent.toLeaf` API can walk inside void elements (for example `+<br>+`). The Core editor’s selection logic, by contrast, does not allow this.
+
+Consequently, when comparing selections where one is from `editor.selection.getRng()` and the other is created by `DomDescent.toLeaf`, the comparison can return `false` when it should be `true`.
+
+When this happens, code runs when it does not need to: in this case, Spell Checker Pro’s ‘preserve selection’ logic.
+
+With this update, SpellcheckerPro now correctly handles `DomDescent.toLeaf` being able to walk into a void element, making the range comparison more accurate, and ensuring the preserve selection code does not run when not needed.
```

Comment:
While I appreciate the technical deep-dive, and this is in fact the bug that we fixed, what it doesn't explain is the relevance to the customer. The customer relevance is:

> Spell Checker Pro could cause inline editing sections to take window focus after scrolling the window. This was particularly evident with multiple inline editors on a single page.

Reply:
Yeah a lot of this is talking about the internals which we don't want discussed in public documentation.

Reply:
Quoting my `git` commit message again:

Re-written Spell Checker Pro section: less technical walk-through, more end-user behaviour change.